src/styles/blocks/_header.scss

```diff
@@ -0,0 +1,40 @@
+.header {
+  background-image: url(../images/header.jpg);
+  background-position: center;
+  background-size: cover;
+  padding: 25px 20px;
+
+  &__container {
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+    height: 100vh;
+    max-width: 1020px;
+    margin: 0 auto;
+  }
+
+  &__title {
+    font-size: 46px;
+    font-weight: 700;
+    letter-spacing: -1px;
+    line-height: 40px;
+    color: #fff;
+    margin: 0;
+    margin-bottom: 80px;
```

Comment:
Better change padding for your outer container, it's not 80 px now https://prnt.sc/1txna6f